test/test_data/test_target_validator.py

```diff
@@ -450,7 +450,7 @@ def test_targetvalidator_inversetransform():
     y_decoded = validator.inverse_transform(y)
     assert ['a', 'a', 'b', 'c', 'a'] == y_decoded.tolist()
 
-    assert validator.classes.tolist() == ['a', 'b', 'c']
+    assert validator.classes_.tolist() == ['a', 'b', 'c']
```

Comment:
Sorry, one more question. According to the type annotation, this is already a list. I guess that the either the type annotation or this check is wrong.

Reply:
Yes it should be np.ndarray to comply with other scikit learn estimators. Sadly mypy ignores numpy (well, treated as ANY) 

There is a typing extension module [here](https://pypi.org/project/typing-extensions/) in case you want to enhance this. For the time being I have moved away from list.

Reply:
We're getting proper annotations in the upcoming releases: https://numpy.org/doc/1.20/release/1.20.0-notes.html

Therefore, we can just keep an eye on this and move to these new types once they are stable and scikit-learn uses them, too.